.github/review-bot.yml

```diff
@@ -0,0 +1,37 @@
+rules:
+  - name: CI Files
+    condition:
+      include:
+        - ^\.github/.*
+    type: basic
+    minFellowsRank: 6
+    min_approvals: 2
+  - name: Relay files
+    condition:
+      include:
+        - ^relay\/kusama\/.*
+        - ^relay\/polkadot\/.*
+      exclude: 
+        - ^relay\/.+\.adoc$
+    type: basic
+    minFellowsRank: 4
+    min_approvals: 2
+  - name: System Parachain Files
+    condition:
+      include:
+        - ^system-parachains\/.*
+    type: basic
+    minFellowsRank: 2
+    min_approvals: 2
```

Comment:
In addition to what @bkchr, the standard to modify system parachain files should not be any lower than that of the Relay Chains.